packages/chat-core/src/index.ts

```diff
@@ -316,6 +330,15 @@ export const createConversation = (config: Config): ConversationHandler => {
     null;
 
   const sendToBot = async (body: BotRequest): Promise<void> => {
+    const lastExpirationTimestamp = getLastExpirationTimestamp(state.responses);
+    if (
+      typeof lastExpirationTimestamp === "number" &&
+      new Date().getTime() > lastExpirationTimestamp
+    ) {
+      setState({
+        conversationId: uuid(),
```

Comment:
curious how this works for a multimodal when someone is already on the phone. Should this fail hard instead of resetting with a new conversation ID? What's the UX here

Reply:
This package does not cover multimodal.

Reply:
oh yeah! whoops